fineract-provider/src/main/java/org/apache/fineract/organisation/staff/service/StaffReadPlatformServiceImpl.java

```diff
@@ -257,14 +252,7 @@ private String getStaffCriteria(final String sqlSearch, final Long officeId, fin
         // employee who does not belong to his office or a sub office for his
         // office.
 
```

Comment:
I think this comment is now in the wrong place and should be moved to retrieveAllStaff(), right?